src/data/matrix/basic.lean

```diff
@@ -397,6 +445,13 @@ def ring_hom.map_matrix [decidable_eq m] [semiring α] {β : Type w} [semiring 
 
 open_locale matrix
 
+/-- Specialize `ring_hom.map_mul` to use `matrix.mul` instead of `has_mul.mul`. -/
+@[simp]
+lemma map_matrix.map_mul [decidable_eq m] [semiring α] {β : Type w} [semiring β]
+  (f : α →+* β) (M N : matrix m m α) :
+  f.map_matrix (M ⬝ N) = f.map_matrix M ⬝ f.map_matrix N :=
```

Comment:
Is this simp-normal? The lemma above simplifies this to
```lean
  (M ⬝ N),map f = M.map f ⬝ N.map f :=
```
doesn't it?

Reply:
The `simp` linter doesn't complain, but it seems to be a problem indeed.